ffc-pay-alerts/notify/notify-template/get-template-id.js

```diff
@@ -0,0 +1,9 @@
+const templateSchema = require('../../schema/event-schema')
+const { notifyEmailTemplateId } = require('../../config')
+
+const getTemplateId = (message) => {
+  const eventMatch = templateSchema.find(x => x.eventType === message?.properties?.action?.type)

```

Comment:
Don't think `message?.properties?.action?.type` is what we're comparing against, here's an example event

{
   "name":"batch-processing-error",
   "properties":{
      "id":"b11ceda6-f473-44b0-b08c-f9659bcd36b2",
      "checkpoint":"ffc-pay-batch-processor-local",
      "status":"error",
      "action":{
         "type":"error",
         "message":"Invalid file - Unknown line",
         "data":{
            "filename":"SITIELM0003_UNKNOWN_LINE_20220307172712493.dat"
         },
         "timestamp":"2022-05-20T15:58:57.938Z"
      }
   }
}

Reply:
With reference to the template content on https://www.notifications.service.gov.uk/services/822632b6-1b07-44ef-9fe0-24dc9bb99272/templates/5e100433-9090-49ad-8f52-25e4751a7544 , the target property is ` message.properties.action.type` .

Reply:
`message.properties.action.type` will only gives us `error` which isn't enough to decide whether we need to forward that onto RPA, platform, etc.

When I created the filter on email address I used the `name` property (in this case `batch-processing-error` to decide which - in my case - email address(es) to use.

Do you think `name` could be used to decide whether `batch-processing-error` uses 1 template versus another?

Reply:
No, I don't think so, name in this context would give further details, hence, I think it should be embedded in the content of the email.